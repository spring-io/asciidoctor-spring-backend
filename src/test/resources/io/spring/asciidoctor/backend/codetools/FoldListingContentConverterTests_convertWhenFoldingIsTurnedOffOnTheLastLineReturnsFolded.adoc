= Test Document

[source,java]
----
// @fold:on
public class Test {

}
// @fold:off
----