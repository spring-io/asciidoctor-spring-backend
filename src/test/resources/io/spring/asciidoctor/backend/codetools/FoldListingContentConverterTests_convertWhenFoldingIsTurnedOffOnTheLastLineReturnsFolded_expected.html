<span class="fold-block hide-when-folded">public class Test {

}
</span>